query.go

```diff
@@ -156,6 +156,10 @@ func getFrames(rows *sql.Rows, limit int64, converters []sqlutil.Converter, fill
 				return nil, err
 			}
 			return frames.Frames(), nil
+		} else {
```

Comment:
Is it possible for the frame type to already be multi, in which case we shouldn't error?

Reply:
hmmm.. good question.. so, `sqlutil.FrameFromRows` returns a single frame... in theory that could already be "multi".. 

i think `sqlutil.FrameFromRows` does not set `frame.type`, so we cannot rely on checking that value.. we would have to somehow manually inspect the frame... 

so.. a multi-frame.. should mean it only has two fields, one of them is a timestamp and the other one is a number, right?